docs/src/maintain/review-pull-requests.md

```diff
@@ -92,16 +92,16 @@ If the pull request was created from a branch on the `eslint/eslint` repository
 There are several times when it's appropriate to close a pull request without merging:
 
 1. The pull request addresses an issue that is already fixed
-1. The pull request hasn't been updated in 30 days
+1. The pull request hasn't been updated in 17 days
 1. The pull request submitter isn't willing to follow project guidelines.
 
 In any of these cases, please be sure to leave a comment stating why the pull request is being closed.
 
 ### Example Closing Comments
 
-If a pull request hasn't been updated in 30 days:
+If a pull request hasn't been updated in 17 days:
```

Comment:
Any particular reason we are using 17 days? Also Im not sure but this feels very less compared to the last duration of 30 days 😅

Reply:
> Any particular reason we are using 17 days?

10 + 7. Bot adds the Stale label and comment after 10 days of inactivity, then closes the PR after another 7 days if there is still no activity.